File: programmers/Lv_1/JeongJunKwon/문자열을 정수로 바꾸기.java
class Solution {
  public int solution(String s) {
      int answer = 0;
      boolean isPlus = true;
      for(int i = 0; i < s.length(); i++) {
          char c1 = s.charAt(i);
          if(c1 == '+')
              continue;
          if(c1 == '-') {
              isPlus = false;
              continue;
          }
          int num = c1 - '0';
          answer = answer * 10 + num;
      }
      
      if(!isPlus)
          answer = -answer;
      return answer;
  }
}

File: programmers/Lv_1/KimHyunseo/문자열을정수로바꾸기.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

int solution(string s) {
	int answer = stoi(s);
	//stoi함수쓰면 int로 변환가능
	return answer;
}

int main()
{
	cout<<solution("1234");
}